node_common/utilities.js

```diff
@@ -6,8 +6,7 @@ import * as Constants from "~/node_common/constants";
 import JWT from "jsonwebtoken";
 import BCrypt from "bcrypt";
 
-import { Buckets } from "@textile/hub";
-import { Libp2pCryptoIdentity } from "@textile/threads-core";
+import { Buckets, PrivateKey } from "@textile/hub";
```

Comment:
PrivateKey uses the same internal key types, the class has some adding signing, encryption, decryption methods for use later